search-app/docs/api.md

```diff
@@ -3418,6 +3418,8 @@ These parameters will match fields within a variable. They are case insensitive
   * options: pattern, ignore_case
 * instrument
   * options: pattern, ignore_case
+* measurement
```

Comment:
Do we have requirement to provide search by measurement? There is no tests for searching by measurements in integration tests. Maybe you want to double checked with system engineering team to confirm that variable measurement is indeed what we have indexed and ideally you should add in the missing tests.

Reply:
I'm not adding tests for a documentation ticket.

Reply:
Measurement is currently supported via search, if we don't want that we should write a ticket to remove it.